website/docs/source/docs/boxes/base_macos.html.md

```diff
@@ -0,0 +1,172 @@
+---
+page_title: "Creating a macOS Base Box Manually"
+sidebar_current: "boxes-macos-base"
+---
+
+# Creating a macOS Base Box Manually
+
+Prior to reading this page, please check out the [basics of the Vagrant
+box file format](https://www.vagrantup.com/docs/boxes/format.html).
+
+## Pre-requisites
+
+* Download and install [Vagrant](https://developer.hashicorp.com/vagrant/downloads).
+* Install Parallels plugin for Vagrant by executing the command below in Terminal:
+  
+    ```bash
+    $ vagrant plugin install vagrant-parallels
+    ```
+
+* Start Parallels Desktop and create a new macOS virtual machine as outlined  in [KB 125561](http://kb.parallels.com/125561/).
+    <div class="alert alert-info">
+    <p>
+        While creating the VM, give the username as ‘vagrant’ only.
+    </p>
+    </div>
+
+  * Enable passwordless sudo for macOS virtual machine
+  * Open Terminal in macOS virtual machine and run:
+
+     ```bash
+     $ sudo visudo
+     ```
+
+    Edit the line:
+
+    ```bash
+    %admin ALL=(ALL) ALL
+    ```
+
+    To Say:
+
+    ``` bash
+    %admin ALL=(ALL) NOPASSWD: ALL
+    ```
+
+    Now, you should now be able to run sudo without password.
+
+* Install Parallels Tools inside the macOS virtual machine.
```

Comment:
a GIF or a link on how to do it would be good. and also that it is essential for vagrant to work and enables copy/paste, shared folders? , prlctl exec

Reply:
Added a gif and aa better description